### ISSUE TITLE
watchexec stopped working or listening on files

### ISSUE THREAD

**Issue by nikivdev** [watchexec/watchexec#459]
Please delete this template text before filing, but you _need_ to include the following:

- Watchexec's version: 1.20.6
- The OS you're using: macOS 13.1
- A log with `-vvv --log-file ../watchexec.log` (if it has sensitive info you can email it at felix@passcod.name — do that _after_ filing so you can reference the issue ID)

Not sure how to get above.  Where is the location of the .log file, I can't find it.

- A sample command that you've run that has the issue

On this repo: https://github.com/nikitavoloboev/ts

If I run:

```
❯ deno run main.ts
test
```

It works. However if I run:

<img width="432" alt="image" src="https://user-images.githubusercontent.com/6391776/209332049-85689cb1-ca82-439e-9510-2d12936bd6d7.png">

Nothing happens. Or I usually run it like:

`watchexec --restart --exts ts "tput reset && deno run main.ts"`

This does nothing too now, not sure why.

**Comment by passcod** [comment]
What do you mean by "now"? Did it work before a certain date? System updates? Watchexec version?

If you run any of these commands with the options `-vvv --log-file ../watchexec.log`, the log file will be one directory up. That's just a simple workaround to avoid the logfile triggering watchexec itself in a loop; get it to write elsewhere if needed.

**Comment by nikitavoloboev** [comment]
Okay I run this: `watchexec -vvv --log-file ../watchexec.log --restart --exts ts "tput reset && bun main.ts"`

On this repo: https://github.com/nikitavoloboev/ts

My log is here: https://gist.github.com/nikitavoloboev/5cad8f4ad1925ad2d97386365f8fc839

The log is quite large so not sure what the error is.

The problem though as I see it is that it's super slow to detect changes. Like 1 minute slow.

<img width="171" alt="image" src="https://user-images.githubusercontent.com/6391776/209548620-47f9a7be-178d-4585-bcc8-035c00b6c533.png">

For the first trigger, after that it seems to work okay. So really not sure what's going wrong.

**Comment by nikitavoloboev** [comment]
As for your commend on when it stopped working, not sure exact moment but before it did work perfectly from the first time and detections were fast + first time I run the command I instantly saw the output.

Watchexec's version: 1.20.6
OS you're using: macOS 13.1

Maybe it happened after I upgraded macOS to latest version.

**Comment by passcod** [comment]
Right, so, this is a pathological case of #377. Set `--project-origin .` and that should disappear. That workaround will stop being necessary by next release, whenever I get around to that.

**Comment by passcod** [comment]
Closing from age and since 377 also closed
